blocks/library/button/index.js

```diff
@@ -171,6 +128,14 @@ class ButtonBlock extends Component {
 	}
 }
 
+const ButtonBlockFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, color } = ownProps.attributes;
+	return {
+		fallbackBackgroundColor: color ? undefined : getComputedStyle( node.querySelector( `.${ ownProps.className }` ) ).backgroundColor,
```

Comment:
Alternatively, you can use `span[key="button"]` as a query selector.

Reply:
Hi @gziolo, I tried this approach, but unfortunately react element keys are not passed to the dom (possible to confirm using browser dev tools), so this query selector was not returning and I end up not applying this change.

Reply:
Fair enough 👍